pkg/webhooks/machine_webhook.go

```diff
@@ -85,7 +85,13 @@ const (
 	defaultAzureCredentialsSecret = "azure-cloud-credentials"
 	defaultAzureOSDiskOSType      = "Linux"
 	defaultAzureOSDiskStorageType = "Premium_LRS"
-	azureMaxDiskSizeGB            = 32768
+
+	// Azure OSDisk constants
+	azureMaxDiskSizeGB                 = 32768
+	azureEphemeralStorageLocationLocal = "Local"
+	azureCachingTypeNone               = "None"
+	azureCachingTypeReadOnly           = "ReadOnly"
+	azureCachingTypeReadWrite          = "ReadWrite"
```

Comment:
Are there constants in the openshift/api we can use here instead?

Reply:
These constants are defined in the Azure SDK, which is not in the dependencies. Probably it doesn't make much sense to import it in this case.